recipes/python/retail/Dockerfile

```diff
@@ -17,14 +17,19 @@
 
 FROM adobe/acp-dsw-ml-runtime-python:0.26.2
```

Comment:
According to this: https://wiki.corp.adobe.com/display/AEPSRE/Dependent+Library+Management#DependentLibraryManagement-MLRuntimePythonCPU
the python runtime is at 0.27.0.  Can you verify there is a good reason why we should get latest here?